packages/components/src/components/carousel/carousel-styles.ts

```diff
@@ -183,24 +190,65 @@ export const getComponentCss = (
     //     display: block,
     //   }
     ...(hasPagination && {
-      pagination: {
+      ['pagination-container']: {
         ...buildResponsiveStyles(hasPagination, (hasPaginationValue: boolean) => ({
           display: hasPaginationValue ? 'flex' : 'none',
         })),
-        justifyContent: 'center',
+        position: 'relative',
```

Comment:
Shouldn't be needed?

Reply:
Positioning is done with left: 'calc(50% - 42px)'